k8s/deploy_app.sh

```diff
@@ -1,15 +1,11 @@
 #!/usr/bin/env bash
-
-set -e
+set -euxo pipefail
 
 if [[ -z "$RUNNER_URL" ]]; then
   echo "RUNNER_URL is mandatory"
   exit 1
 fi
```

Comment:
This should now either check all DOCKER_REGISTRY, IMAGE_TAG and RUNNER_URL is set or we should remove all. The script doesn't seem to be very useful now.

Reply:
It can go - the `u` will treat an unset variable as an error

Reply:
We should still use a sh file over the yaml

Reply:
Values can be marked as required in Helm, which we should do (in another pull request).

Reply:
I'll add a card